### DISCUSSION TITLE
How to get access_token and refresh_token for the target channel(which the video will be uploaded to)

### DISCUSSION THREAD

**Discussion by chen-zhuofu** [pillargg/youtube-upload#21] [Q&A]
Tried OAuth 2.0 Playground for the target channel.
Get this error:
googleapiclient.errors.ResumableUploadError: <HttpError 403 when requesting None returned "The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.". Details: "[{'message': 'The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.', 'domain': 'youtube.quota', 'reason': 'quotaExceeded'}]">

Ask after I checked to make sure that I have enough quota.
And when switch back to authenticate() [no params] way. It works fine

**Comment by yeoengheng** [comment]
Would love to know the answer to this too
